ocdaction/ocdaction/templates/challenge/challenge_view.html

```diff
@@ -4,6 +4,11 @@
 
 {% block main %}
     <section class="container">
+        <div class="col-xs-offset-3">
+            <a href="{% url 'challenge-list' %}">
```

Comment:
A minor tweak: can we go back to the archive page if that is an archived challenge page? (sorry if I missed that in the story description!)

Reply:
No problem, it was a quick tweak to do!